### DISCUSSION TITLE
BleachBit cannot read contents of folder

### DISCUSSION THREAD

**Discussion by ghost** [bleachbit/bleachbit#1234] [Q&A] [answered]
Hi! I'm new to BleachBit so I hope the questions aren't trivial but I couldn't find an answer anywhere:

1. BleachBit cannot read the contents of '%windir%\system32\sru'. Why and how do I change that?
    (I know that admin rights are necessary to access that folder but BleachBit has them, right? Every other elevated software I tried can do it. Deletion needs additional measures but that's not a problem.)
2. How do I escape a slash in a json address?

**Comment by ghost** [comment, marked as answer]
For reference:

1. The solution is in issue [#1240](https://github.com/bleachbit/bleachbit/issues/1240).
2. It's currently impossible. "clean_json" just uses "target.split('/')" where target is the json address.

**Reply by az0** [reply]
I am glad you figured these out (in a way), though there is more work to do.
